opm/autodiff/FullyImplicitBlackoilSolver.hpp

```diff
@@ -196,6 +196,9 @@ namespace Opm {
 
         LinearisedBlackoilResidual residual_;
 
+        /// \brief Wether we print something to std::cout
+        bool verbosity_;
```

Comment:
To me `verbosity_` implies that there are some levels, not just a bool. I suggest changing the name to `terminal_output_`.

Also minor typo: wether -> whether (wether means a male sheep...)